Command/VersionBumpCommand.php

```diff
@@ -62,44 +62,42 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
             $formatter = $this->manager->getFormatter();
             if ($formatter instanceof FormatterInterface) {
-                $output->writeln(sprintf('Formatter: <comment>%s</comment>', get_class($formatter)));
+                $textOutput = get_class($formatter);
```

Comment:
Actually I cannot see a benefit of assigning the formatter type to a variable here. So revert this change or convince me that it makes sense :-)

Reply:
I reverted it.

I have done that to avoid a repetition: `'Formatter: <comment>%s</comment>'`.